### PR TITLE
LMS-163

### PR DESCRIPTION
**Pull request by OlehKosylo**
Fixed bag with creating modules

### CODE REVIEW

File: src/interfaces/lesson.model.ts [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
-    tags?: [string];
-    module_id: string;
+  _id?: string;
+  number: string | number;
+  label: string;
+  description: string;
+  video_path: string;
+  tags: [string];
+  module_id: string;
+  questions_id?: [string];
```
Comment (VictorKmin) [inline src/interfaces/lesson.model.ts:9]: Why do you use it there?
We have a `question -> lesson` relation. We don't need a `lesson -> question`

File: src/validators/module/moduleValidator.ts [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
-import { lessonValidator } from '../lesson';
-
 export const moduleValidator = Joi.object().keys({
   label: Joi.string().max(255).trim().required(),
   description: Joi.string().max(255).trim(),
-  tags: Joi.array().items(Joi.string()),
+  tag_list: Joi.array().items(Joi.string()),
   courses_id: Joi.array().items(Joi.string()),
-  lessons_list: Joi.array().items(lessonValidator)
+  lessons_list: Joi.array().items(Joi.string())
```
Comment (VictorKmin) [inline src/validators/module/moduleValidator.ts:8]: Why did you removed 
```js
 import { lessonValidator } from '../lesson';
```

File: src/services/lesson/lesson.service.ts [hunk 3]
```diff
@@ -1,4 +1,6 @@
-import { Lesson } from '../../database';
+import { model } from 'mongoose';
+import { DatabaseTablesEnum } from '../../constants/enums';
```
Comment (VictorKmin) [inline src/services/lesson/lesson.service.ts:2]: line

File: src/routes/lesson/lesson.router.ts [hunk 4]
```diff
@@ -1,9 +1,11 @@
 import { Router } from 'express';
 
 import { lessonController } from '../../controllers';
+import { isLessonFilterValid } from '../../middleware/lesson';
```
Comment (VictorKmin) [inline src/routes/lesson/lesson.router.ts:4]:
```js
import { isLessonFilterValid } from '../../middleware';
```

File: src/validators/module/moduleValidator.ts (hunk 2 above)
Comment (OlehKosylo) [reply src/validators/module/moduleValidator.ts:8]: Because it didn't work

File: src/services/lesson/lesson.service.ts (hunk 3 above)
Comment (OlehKosylo) [reply src/services/lesson/lesson.service.ts:2]: Done

File: src/interfaces/lesson.model.ts (hunk 1 above)
Comment (OlehKosylo) [reply src/interfaces/lesson.model.ts:9]: My mistake, fixed

**Pull request merged**